.changeset/silent-garlics-fetch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ai-sdk/azure': patch
+---
+
+azure responses image-generation tool streaming is enabled.
```

Comment:
if only docs and tests are updated, no `.changeset` file needs to be created, since no code that runs in production has been changed.